src/lua/packagepath.lua

```diff
@@ -0,0 +1,50 @@
+-- packagepath.lua
+--
+-- The purpose of package.setpath() is to be called from the
+-- application entry point script to make relative imports and imports
+-- from .rocks subdirectory possible. package.setpath() is recommended
+-- to be called from entrypoint before any require() calls.
+--
+-- Without this if the app is started while current working directory
+-- is not application root, relative require() and attempts to
+-- require() lua packages installed to .rocks will fail, because all
+-- require() calls will load modules relatively to current working
+-- directory.
+
+local fio = require('fio')
+
+local function extend_path(path)
+    package.path = package.path .. ';' .. path
```

Comment:
We can't expect package.path to end with semicolon, because for all we know the user could have added anything to the right part of package.path and not terminate it with said semicolon. I could've done string.rstrip(package.path, ";"), but we don't have it implemented, and I didn't want to complicate the code to evade the extra symbol.

If you insist, I'd make an effort to clean it up.